libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/routeprogress/RouteProgress.java

```diff
@@ -250,6 +279,8 @@ public RouteProgress build() {
   }
 
   public static Builder builder() {
-    return new AutoValue_RouteProgress.Builder();
+    return new AutoValue_RouteProgress.Builder()
+      .currentAnnouncement(EMPTY_ANNOUNCEMENT)
```

Comment:
Saw that you removed the explicitly `inTunnel` initialization here. That would mean that there's no a `false` `inTunnel` default value anymore (making it a "required" property) so clients using `RouteProgress` will need to initialize it explicitly (same as we did with these latest changes) if they don't want to get a `IllegalStateException`. Is that what we want? I'm down with either but I wanted to confirm that we were on the same page.